great_expectations_cloud/agent/actions/generate_data_quality_check_expectations_action.py

```diff
@@ -271,6 +271,10 @@ def _add_completeness_change_expectations(
                 options = TriangularInterpolationOptions(
                     input_range=(0.0, float(row_count)), output_range=(0, 0.1), round_precision=5
                 )
+                if null_count is None:
```

Comment:
I think we should move this up into another `elif` and not make any expectations in this case. `None` indicates not rows in the table so we don't have any expectations about nullity.